src/test/java/org/jpeek/AppTest.java

```diff
@@ -111,4 +116,29 @@ public void createsIndexXml() throws IOException {
             )
         );
     }
+
+    @Test
+    public void isXsdDocumented() throws IOException {
+        final List<XML> elements = new XMLDocument(
+            AppTest.class.getResourceAsStream("xsd/metric.xsd")
+        ).nodes("//node()[@name]");
+        final IsNot<? super List<?>> populated = new IsNot<>(
```

Comment:
@ilyakharlamov I don't think that this is necessary here, actually creating this variable just makes more difficult to understant what are we trying to match in the assertions. You could just use `new IsNot<>(new IsEmptyIterable<>())` twice

Reply:
Otherwise, I will have to use creating `new IsNot<>(new IsEmptyIterable<>())` in the loop.
"new" in the loop is also considered an anti-pattern usually.